### DISCUSSION TITLE
ath10k-ct-2021-11-21-e8e166fe.tar.xz的hash code应该是错的

### DISCUSSION THREAD

**Discussion by zcluo** [Boos4721/openwrt#51] [Bugs/问题反馈]
time: package/lean/luci-app-turboacc/compile#0.28#0.11#0.38
ath10k-ct-2021-11-21-e8e166fe.tar.xz: Download from https://github.com/greearb/ath10k-ct.git failed
ath10k-ct-2021-11-21-e8e166fe.tar.xz: Wrong hash (probably caused by .gitattributes), expecting f455df42e42c49381cbc3b8e5b510d877e2f9974641cccb73408f0c5b96c34ce, got 771193292242fd8c198e9e379aa90882b1f5a144fbd77e09fbc97aca8ffe84e1
Checking out files from the git repository...
Cloning into 'ath10k-ct-2021-11-21-e8e166fe'...

**Comment by Boos4721** [comment]
**FIXED**
